support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -58,6 +58,7 @@ export const tests: Tests = {
 		isActive: true,
 		referrerControlled: true,
 		seed: 2,
+		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
```

Comment:
Utterly minor but would is be better to label this `allContributionsLandingAndThankyouPages` or is the wider description sufficient.